assets/src/scss/new-identity/style.scss

```diff
@@ -122,27 +136,23 @@ table.spreadsheet-table.is-color-gp-green {
   }
 }
 
-.standalone-link {
-  width: fit-content;
-  font-weight: 700 !important;
-}
-
 // Body Copy links selectors
 .post-content,
 .page-content {
-  .standalone-link,
   p > a,
   li > a {
-    transition: 0.2s cubic-bezier(0.5, 0, 0, 1);
-    border-bottom: 2px solid var(--gp-green-500);
-
-    &:hover {
-      box-shadow: inset 0 calc(-22px) var(--gp-green-100);
-      text-decoration: none !important;
-    }
+    @include shared-link-styles;
   }
 }
 
+.standalone-link,
```

Comment:
Instead of using a mixin, we could also just have kept these two under the `.post-content, .page-content` selector 🤔 I can't think of a case where they would be somewhere else?

Reply:
But the comments block is not into the page|post content

![Uploading Screenshot 2023-04-26 at 08.59.19.png…]()